packages/react/src/primitives/SwitchField/useSwitch.ts

```diff
@@ -20,7 +28,10 @@ export const useSwitch = (props: UseSwitchProps) => {
         return;
       }
 
-      typeof onChange === 'function' && onChange(event);
+      if (typeof onChange === 'function') {
```

Comment:
We have a `isFunction` util under `shared` folder

Reply:
Really trying not to get in to refactoring in these PRs to prevent scope creep from kicking in